### ISSUE TITLE
A11y_AzureToolsStorageExplorer_SharedAccessSignature_Screenreader:NVDA does not announce the value present inside the Show month selection panel button.

### ISSUE THREAD

**Issue by Charanrajkb** [microsoft/AzureStorageExplorer#1914]
" [Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing aster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA; Win32; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Blob gets uploaded, right click on blob and select "Get Shared Access Signature" control and hit enter
7. Start NVDA and navigate using the tab key till the "Show Date picker" and expand it.
8. Observe the NVDA readings.

​**Actual:​**
NVDA does not announce the value present inside the Show month selection panel button.
NVDA reads the name role but it does not announce the value present.

**Expected:**
Screen reader must read the name role and value for the controls.
NVDA must read the value as "Show month panel September 2019 selected button"

**User Impact:**
Screen readers user will not understand the Value present for  the control.
Screen reader users will not come to know which month and year has been selected.

**Recommendation:**
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**MAS Reference:**
MAS1.3.1:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment for Reference:**
[1914_A11y_AzureToolsStorageExplorer_SharedAccessSignature_Screenreader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3652417/1914_A11y_AzureToolsStorageExplorer_SharedAccessSignature_Screenreader.pptx)

**Comment by jinglouMSFT** [comment]
@Charanrajkb Can you guys please check our VPAT document for known external issues or just known issues before opening bugs?  There is no point for us to triage the known external issues where we have already opened issues on the 3rd party.

**Comment by craxal** [comment]
See https://bugs.chromium.org/p/chromium/issues/detail?id=761552&q=761552&can=4

**Comment by MRayermannMSFT** [comment]
Make sure to add to VPAT

**Comment by Alexandria-R** [comment]
Approved due to workaround

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
